### DISCUSSION TITLE
Feed for All Newly Updated Dockets (Any Type, etc.)

### DISCUSSION THREAD

**Discussion by becker-charles** [freelawproject/courtlistener#3161] [Q&A] [answered]
Any better way to get this? The below seems to work, but it is very slow (likely putting lots of strain on the server).

`https://www.courtlistener.com/feed/search/?type=r&order_by=dateFiled+desc&available_only=on`

**Comment by mlissner** [comment]
In English what are you trying you accomplish?

**Comment by becker-charles** [comment]
Get a feed (does not need to be RSS, I suppose) that provides the dockets that were most recently updated. No constraint on the cause of action, parties, etc. Just all recently updated dockets (e.g., a new PDF has been uploaded on PACER). Thank you for your help on this.

**Comment by becker-charles** [comment]
I was further mistaken about this. The RSS feed link I am using `https://www.courtlistener.com/feed/search/?type=r&order_by=dateFiled+desc&available_only=on` does not get them as they are updated, it seems. Serves the same content for hours on end.

**Comment by mlissner** [comment, marked as answer]
So it might be hard to do this as an RSS feed, I'm afraid. We don't even have a field in the database that keeps track of *when* a PDF is added to a case, though we do have a flag for whether there's a PDF. 

I think if I wanted to do what you're describing, I'd use the API. That doesn't give you an RSS feed, but it would allow you to do an ordered result based on date_modified, and you could use that to figure out which ones have documents. Could that work? 

The other option is to wait for our new search engine, which we're working on now. It will have an alert system that will trigger emails or webhooks whenever a query has new results. If all you wanted is for a server to know when there are new results for a given query, you could use the webhooks to do so (we'll probably charge commercial organizations for this, though). When this is launched, we'll do a blog post. If you want a github issue to follow, you could follow this one: https://github.com/freelawproject/free.law/issues/124

**Reply by becker-charles** [reply]
I think the API could work if I understand you right. Will need to research how to get access, but I think your post solves this discussion post. 

For reference, I am a student trying to continuously pull/analyze newly filed court documents in mass for research. RECAP seems to be the best source I can find (at least without getting a small loan of a million dollars to pay our beloved government).

**Reply by mlissner** [reply]
That's great. Yeah, check out the API, and let us know what new questions it provokes!
